sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLLibraryDAO.java

```diff
@@ -387,7 +387,8 @@ public long save(Library library) throws IOException {
           String name = libraryNamingScheme.generateNameFor("name", library);
           library.setName(name);
           if (libraryNamingScheme.validateField("name", library.getName())
-              && libraryNamingScheme.validateField("alias", library.getAlias())) {
+              && (libraryNamingScheme.validateField("alias", library.getAlias())
+                  || library.getLibraryAdditionalInfo().hasNonStandardAlias())) {
```

Comment:
* Needs null check on LibraryAdditionalInfo
* Same below (L420)
* Above (L380), there is a check for duplicates, which I think should check the nonStandardAlias attribute as well